tests/requirements.txt

```diff
@@ -1,4 +1,5 @@
 paramiko
 msgpack
 loguru
-coincurve
\ No newline at end of file
+coincurve
+iso3166
```

Comment:
Do we still need this?

Reply:
we need the iso3166. Checking about the coincurve